TYPO3.Neos/Documentation/ExtendingNeos/InteractionWithTheNeosBackend.rst

```diff
@@ -52,6 +53,17 @@ events occur:
 * **Neos.FullScreenModeActivated** When the backend switches to fullscreen mode.
 * **Neos.FullScreenModeDeactivated** When the backend leaves the fullscreen mode.
 
+Example of interacting with the selected node element using the ``NodeSelected`` event.
+
+.. code-block:: javascript
+
+  document.addEventListener('Neos.NodeSelected', function(event) {
+  	var node = event.detail.node;
+  	if (event.detail.node.get('nodeType') === 'Acme:Demo') {
+  		console.log(node.getAttribute('title'), node.get('attributes.title'), node.$element);
```

Comment:
I get an error with this example: `node.getAttribute is not a function`
But `console.log(node.get('attributes.title'), node.$element);` works.